images/macos/provision/utils/utils.sh

```diff
@@ -240,7 +192,18 @@ get_github_package_download_url() {
 
     [ -n "$API_PAT" ] && authString=(-H "Authorization: token ${API_PAT}")
 
-    json=$(curl "${authString[@]}" -fsSL "https://api.github.com/repos/${REPO_ORG}/releases?per_page=${SEARCH_IN_COUNT}")
+    failed=true
+    for i in {1..10}; do
+        curl "${authString[@]}" -fsSL "https://api.github.com/repos/${REPO_ORG}/releases?per_page=${SEARCH_IN_COUNT}" >/tmp/get_github_package_download_url.json && failed=false || sleep 60
+        [ "$failed" = false ] && break
+    done
+
+    if [ "$failed" = true ]; then
+       echo "Failed: brew --cache $dep"
```

Comment:
Seems like you forgot to update error message text here

Reply:
thanks, fixed